tests/testthat/test-label-dollar.R

```diff
@@ -25,15 +25,15 @@ test_that("rescale_large works", {
   x <- 10^(seq(0, 18, 3))
```

Comment:
We need to rename the 2 testthat blocks because they reference the `rescale_large` argument.